src/js/components/Select/index.d.ts

```diff
@@ -8,6 +8,7 @@ export interface SelectProps {
   gridArea?: GridAreaType;
   children?: ((...args: any[]) => any);
   closeOnChange?: boolean;
+  clear?: boolean | { position?: "top" | "bottom", label?: string, renderValue?: ((param: {onClear: () => void }) => React.ReactNode)};
```

Comment:
```suggestion
    clear?: boolean | { position?: "top" | "bottom", label?: string, renderValue?: ((param: {onClear: () => void }) => React.ReactNode)};
  closeOnChange?: boolean;
```
maintaining alphabetical order.